exec-unittests

```diff
@@ -131,7 +131,10 @@ if [[ -z "${subtst}" || "${subtst}" == "python" ]]; then
     for _major in ${major_list}; do
         _pytest_majors="${_pytest_majors} pbench.test.unit.${_major}"
         if [[ "${_major}" == "agent" ]]; then
-            _pytest_majors="${_pytest_majors} pbench.test.functional"
+            # TODO: real functional tests require a deployed instance. Current
+            # agent "functional" tests are mocked Click tests rather than true
+            # "functional" tests.
+            _pytest_majors="${_pytest_majors} pbench.test.functional.agent"
```

Comment:
I have always been annoyed that the click framework tests were termed "functional" when they're not; but now the *agent* is running the server functional tests.

They all "skip" because there's no environment variable, so it's more ugly than disastrous, but, yeah, I guess I feel fairly strongly about omitting *functional* server tests from the *agent* unit test run.

Reply:
My bad:  I failed to pick up on the fact that you'd changed `pbench.test.functional` to `pbench.test.function.agent`!

Nevermind....